packages/vx-glyph/src/glyphs/GlyphCircle.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import cx from 'classnames';
+import { Symbol, symbol, symbolCircle } from 'd3-shape';
+import Glyph from './Glyph';
+
+export type GlyphCircleProps<Datum> = {
+  /** Render function override which is passed the configured path generator. */
+  children?: ({ path }: { path: Symbol<any, Datum> }) => React.ReactNode;
+  /** classname to apply to glyph path element. */
+  className?: string;
+  /** Top offset to apply to glyph g element container. */
+  top?: number;
+  /** Left offset to apply to glyph g element container. */
+  left?: number;
+  /** Size of circle in px, or an accessor which takes Datum as input and returns a size. */
+  size?: number | ((d: Datum) => number);
+};
+
+export default function GlyphCircle<Datum = any>({
```

Comment:
the `Datum` generic here is not useful unless the consumer leverages the `children` override. but since `d3`s `symbol` takes it as an arg I think exposing it is good.